src/wp-includes/option.php

```diff
@@ -392,16 +392,16 @@ function wp_set_option_autoload_values( array $options ) {
 	}
 
 	$grouped_options = array(
-		'yes' => array(),
-		'no'  => array(),
+		'on'  => array(),
+		'off' => array(),
 	);
 	$results         = array();
 	foreach ( $options as $option => $autoload ) {
 		wp_protect_special_option( $option ); // Ensure only valid options can be passed.
 		if ( 'no' === $autoload || false === $autoload ) { // Sanitize autoload value and categorize accordingly.
```

Comment:
We should probably support the new `off` here too, otherwise this may end up with confusing results.
```suggestion
		if ( 'off' === $autoload || 'no' === $autoload || false === $autoload ) { // Sanitize autoload value and categorize accordingly.
```